tests/test_e2e_21_flow_manager.py

```diff
@@ -138,3 +138,64 @@ def test_031_on_switch_restart_kytos_should_recreate_flows(self):
         flows_s1 = s1.dpctl('dump-flows')
         assert len(flows_s1.split('\r\n ')) == BASIC_FLOWS + 1, flows_s1
         assert 'dl_vlan=999' in flows_s1
+
+    def test_032_deserialization_by_restarting_kytos(self):
+        """Test if, after kytos restart, deserialize properly"""
+
+        payload = {
+            "flows": [
+                {
+                    "priority": 10,
+                    "match": {
+                        "in_port": 1,
+                        "dl_vlan": "4096/4096"
+                    },
+                    "actions": [
+                        {
+                            "action_type": "output",
+                            "port": 2
+                        }
+                    ]
+                },
+                {
+                    "priority": 10,
+                    "match": {
+                        "in_port": 1,
+                        "dl_vlan": 0
```

Comment:
Just a heads up regarding this PR https://github.com/kytos-ng/flow_manager/pull/135 and https://github.com/kytos-ng/kytos-end-to-end-tests/pull/209, that `dl_vlan` on flow_manager eventually will expected to be an str even for cases without a mask, could you also update here to be a str?

Reply:
Requirements have changed. `flow_manager` is now accepting int and strs